controllers/configurationpolicy_controller.go

```diff
@@ -1058,12 +1080,23 @@ func (r *ConfigurationPolicyReconciler) handleSingleObj(
 	exists bool,
 	dclient dynamic.Interface,
 	objectT *policyv1.ObjectTemplate,
-) (objNameList []string, compliance bool, rsrcKind string, statusUpdateNeeded bool) {
+) (
+	objNameList []string,
+	compliance bool,
+	rsrcKind string,
+	statusUpdateNeeded bool,
+	creationInfo policyv1.ObjectProperties,
```

Comment:
I'm surprised this doesn't need to return a pointer to identify the difference between nil and the empty values... but I don't think it's worth worrying over it while we're still working on the initial implementation. If it needs changing, I think it will pop up at some point during tests.

Reply:
you're right, this did need to be fixed - it was causing inform policies to have the same properties as enforce policies that did not create the object